packages/material-ui/src/Popper/Popper.test.js

```diff
@@ -234,12 +238,12 @@ describe('<Popper />', () => {
       const ref1 = React.createRef();
       const ref2 = React.createRef();
       const { setProps } = render(<Popper {...defaultProps} popperRef={ref1} />);
-      expect(ref1.current instanceof PopperJs).to.equal(true);
```

Comment:
Popper uses snapshot tests for these. Since the instance is just a plain object now.